docs-2.0/2.quick-start/5.start-stop-service.md

```diff
@@ -121,25 +123,23 @@ Run the following command to check the service status of Nebula Graph.
 $ sudo /usr/local/nebula/scripts/nebula.service status all
 ```
 
-* Nebula Graph is running normally if the following information is returned.
+* Nebula Graph service is running normally if the following information is returned.
```

Comment:
Service is not necessary here. If you wish to keep it, at least use services because there are 3 different services in the following code block.